usaspending_api/download/v2/request_validations.py

```diff
@@ -91,6 +96,52 @@ def validate_idv_request(request_data):
     }
 
 
+def validate_contract_request(request_data):
+    _validate_required_parameters(request_data, ["award_id"])
+    award_id, piid, fain, uri, generated_unique_award_id = _validate_award_id(request_data)
+
+    return {
+        "account_level": "treasury_account",
+        "download_types": ["contract_sub_awards", "contract_transactions", "contract_federal_account_funding"],
+        "file_format": request_data.get("file_format", "csv"),
+        "include_file_description": {
+            "source": settings.CONTRACT_DOWNLOAD_README_FILE_PATH,
+            "destination": "readme.txt",
```

Comment:
Same issue as the Assistance download below. AC says it should be `ContractAwardSummary_download_readme.txt` instead of `readme.txt`.